.circleci/config.yml

```diff
@@ -29,6 +34,13 @@ jobs:
 
 workflows:
   version: 2
-  build:
+  build-and-deploy:
     jobs:
       - build
+      - gcp-gcr/build-and-push-image:
+          requires:
+              - build
+          image: pensieve
+          filters:
+              branches:
+                  only: master
```

Comment:
It seems like this change uses 4 spaces for indentation while the rest of the file uses 2. I'm fine with either (though 2 seems to be the standard?)